assets/airports/engm.json

```diff
@@ -13,825 +13,1920 @@
+                ["N60.710030900", "E10.580539700"],
+                ["N60.709162400", "E10.580585200"],
+                ["N60.708293900", "E10.580537900"],
+                ["N60.707427800", "E10.580398100"],
+                ["N60.706566400", "E10.580166200"],
+                ["N60.705712200", "E10.579842700"],
+                ["N60.704867200", "E10.579428600"],
+                ["N60.704034000", "E10.578924900"],
+                ["N60.703214800", "E10.578333000"],
+                ["N60.702411800", "E10.577654500"],
+                ["N60.701627100", "E10.576891200"],
+                ["N60.700862900", "E10.576045200"],
+                ["N60.700121400", "E10.575118700"],
+                ["N60.699404500", "E10.574114300"],
+                ["N60.698714200", "E10.573034600"],
+                ["N60.698052500", "E10.571882500"],
+                ["N60.697421000", "E10.570661200"],
+                ["N60.696821700", "E10.569374100"],
+                ["N60.696256100", "E10.568024400"],
+                ["N60.695725800", "E10.566616100"],
+                ["N60.695232400", "E10.565153000"],
+                ["N60.694777000", "E10.563638800"],
+                ["N60.694361100", "E10.562078000"],
+                ["N60.693985800", "E10.560474600"],
+                ["N60.693652100", "E10.558833200"],
+                ["N60.693361000", "E10.557158200"],
+                ["N60.693113300", "E10.555454300"],
+                ["N60.692909700", "E10.553726100"],
+                ["N60.692750700", "E10.551978300"],
+                ["N60.692636800", "E10.550215900"],
+                ["N60.692568300", "E10.548443700"],
+                ["N60.692545500", "E10.546666700"],
+                ["N60.692568300", "E10.544889600"],
+                ["N60.692636800", "E10.543117400"],
+                ["N60.692750700", "E10.541355000"],
+                ["N60.692909600", "E10.539607300"],
+                ["N60.693113300", "E10.537879100"],
+                ["N60.693361000", "E10.536175200"],
+                ["N60.693652100", "E10.534500200"],
+                ["N60.693985800", "E10.532858700"],
+                ["N60.694361100", "E10.531255300"],
+                ["N60.694777000", "E10.529694500"],
+                ["N60.695232400", "E10.528180300"],
+                ["N60.695725800", "E10.526717200"],
+                ["N60.696256100", "E10.525308900"],
+                ["N60.696821700", "E10.523959300"],
+                ["N60.697421000", "E10.522672100"],
+                ["N60.698052500", "E10.521450800"],
+                ["N60.698714200", "E10.520298800"],
+                ["N60.699404500", "E10.519219100"],
+                ["N60.700121400", "E10.518214600"],
+                ["N60.700862900", "E10.517288200"],
+                ["N60.701627100", "E10.516442100"],
+                ["N60.702411800", "E10.515678800"],
+                ["N60.703214800", "E10.515000300"],
+                ["N60.704034000", "E10.514408500"
```

Comment:
No speed restriction here at `GM440`, it appears that this restriction only applies to NUVSA9B.